### PR TITLE
83660 update medicare section copy - form 10-7959c

### PR DESCRIPTION
**Pull request by michaelclement**
## Summary

This PR updates copy and adds two new screens to the Medicare section of form 10-7959c.

- Adds new medicare class question
- Adds new medicare additional comment screen
- Fixes end to end tests for this form
- Removes unused unit test for this form
- Removes screen we no longer need
- Makes several small copy tweaks (hints, titles, etc)
- Team: IVC CHAMPVA
- Flipper: NA

## Related issue(s)

- https://github.com/department-of-veterans-affairs/va.gov-team/issues/83660

## Testing done

- Manual
- Unit
- E2E

## Screenshots

|         |  After |
| ------- |  ----- |
| S1 ||
| S2 ||
|S3||
| S4 ||
|S5||

## What areas of the site does it impact?

This form only

### Quality Assurance & Testing

- [X] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [X] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [X] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [X] Browser console contains no warnings or errors.
- [X] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:

- [ ] The vets-website header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

## Requested Feedback

NA

### CODE REVIEW

File: src/applications/ivc-champva/10-7959C/chapters/medicareInformation.js
Comment (michaelclement) [inline src/applications/ivc-champva/10-7959C/chapters/medicareInformation.js:1]: Schema re-arranging. This is the bulk of the non-testing work. Re-ordered some screens, changed wording, and added/removed a couple screen configs.

File: src/applications/ivc-champva/10-7959C/config/form.js
Comment (michaelclement) [inline src/applications/ivc-champva/10-7959C/config/form.js:1]: Pulled in the new re-arranged schemas, deleted unused ones, and adjusted most of the `nameWording` function calls with different arguments (so we only show first names in titles)

File: src/applications/ivc-champva/10-7959C/config/submitTransformer.js
Comment (michaelclement) [inline src/applications/ivc-champva/10-7959C/config/submitTransformer.js:1]: Dropped `applicantMedicareAdvantage` dedicated key, but backend still expects it when filling PDF. Add it here by getting value of `applicantMedicareClass`.

File: src/applications/ivc-champva/10-7959C/helpers/utilities.js [hunk 1]
```diff
@@ -31,7 +31,7 @@ export function nameWording(
   }
 
   // Optionally capitalize first letter and return
-  return cap ? retVal.charAt(0).toUpperCase() + retVal.slice(1) : retVal;
+  return cap ? retVal?.charAt(0)?.toUpperCase() + retVal?.slice(1) : retVal;
```
Comment (michaelclement) [inline src/applications/ivc-champva/10-7959C/helpers/utilities.js:34]: More robust

File: src/applications/ivc-champva/10-7959C/tests/config/form.unit.spec.jsx
Comment (michaelclement) [inline src/applications/ivc-champva/10-7959C/tests/config/form.unit.spec.jsx:1]: dropped unused unit test

File: src/applications/ivc-champva/10-7959C/tests/e2e/fixtures/data/applicant-maximal-test.json
Comment (michaelclement) [inline src/applications/ivc-champva/10-7959C/tests/e2e/fixtures/data/applicant-maximal-test.json:1]: Lots of small changes had to be made across all the E2E data files. All the JSON changes are pretty similar, just dropping data for pages that are no longer conditionally rendered in a given test, and switching some values from strings to booleans.

**Review by balexandr — APPROVED** [review]

**Review by cloudmagic80 — APPROVED** [review]

**Review by ataker — APPROVED** [review]

**Pull request merged**